commune_app/models.py

```diff
@@ -1,3 +1,32 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.core.validators import RegexValidator
 
-# Create your models here.
+
+phone_regax = RegexValidator(regex=r'^\+?1?\d{9,15}$', message="phone number format: '+999999999'")
+
+
+class User(AbstractUser):
+    id = models.IntegerField(unique=True, primary_key=True)
```

Comment:
likewise, we'd prefer to have a docstring
flake8 should be able to enforce this too

Reply:
can you explain more?